test/dataLabel/dataLabelManagerTest.ts

```diff
@@ -31,6 +31,7 @@ module powerbi.extensibility.utils.chart.dataLabel.test {
     import RectOrientation = powerbi.extensibility.utils.chart.dataLabel.RectOrientation;
     import ContentPositions = powerbi.extensibility.utils.chart.dataLabel.ContentPositions;
     import OutsidePlacement = powerbi.extensibility.utils.chart.dataLabel.OutsidePlacement;
+    import LabelPosition = powerbi.extensibility.utils.chart.dataLabel.LabelPosition;
```

Comment:
Why is there an import that is not used anywhere?

Reply:
Changed the way of importing.

This is the only test in a module, and since the namespace conflict with `powerbi`, I had to change the module name.